test/installPSRP.sh

```diff
@@ -17,10 +17,10 @@ chmod +x ./installpowershell.sh
 
 redmondpassword=$1
 isMacOS=false
-omiversion="1.1.0-28"
+omiversion="1.1.0-52"
```

Comment:
Do you have to update this file ever time the versions change? Could you pass the versions in as arguments when running the script?